components/autofill/src/sync/address/incoming.rs

```diff
@@ -266,18 +275,17 @@ fn get_incoming_record_states(conn: &Connection) -> Result<Vec<IncomingState>> {
     Ok(conn
         .conn()
         .query_rows_and_then_named(sql_query, &[], |row| -> Result<IncomingState> {
-            // XXX - change these to something like `mirror_exists` - that's how they are used.
-            let mirror_guid: Option<SyncGuid> = row.get_unwrap("m_guid");
-            let local_guid: Option<SyncGuid> = row.get_unwrap("l_guid");
-            let tombstone_guid: Option<SyncGuid> = row.get_unwrap("t_guid");
+            let mirror_exists: Option<SyncGuid> = row.get_unwrap("m_guid");
```

Comment:
Sorry, I missed this yesterday, but in that comment I meant that `mirror_exists` etc should become a bool as the guid is never actually used - ie, something like `mirror_exists = row.get_unwrap("m_guid").is_some();`- that doesn't work for the tombstone though because we actually use the guid.